src/meta/src/barrier/info.rs

```diff
@@ -197,30 +155,19 @@ impl InflightActorInfo {
                             .expect("should exist");
                         let actors = &mut info.actors;
                         for (actor_id, node_id) in new_actors {
-                            assert!(to_add
-                                .insert(actor_id, (node_id, info.is_injectable))
-                                .is_none());
+                            assert!(to_add.insert(actor_id, node_id).is_none());
```

Comment:
Could you elaborate a bit more on `is_injectable`, I see its use removed in various places, but I don't know what it was used for previously, and why it's obsolete now.

Reply:
The `is_injectable` was to maintaining the `actor_map_to_send`, which is removed in this PR.

The `actor_map_to_send` means the actor that directly receives the barriers from barrier sender on `LocalBarrierWorker` instead of receiving from upstream.